cf-agent/acl_posix.c

```diff
@@ -339,15 +341,33 @@ static int CheckPosixLinuxACEs(EvalContext *ctx, Rlist *aces, AclMethod method,
 
             if (!DONTDO)
             {
+                int last = -1;
+                acl_text_str = acl_to_any_text(acl_new, NULL, ',', 0);
+                Log(LOG_LEVEL_DEBUG, "ACL: new acl is `%s'", acl_text_str);
+
+                if ((retv = acl_check(acl_new, &last)) != 0)
+                {
+                    Log(LOG_LEVEL_ERR, "ACL: acl_check(%s): %s at entry %d. Refer to acl_check man page for validity rules.",
```

Comment:
Try to improve indentation and wrap better, if possible at column 78 max. And make sure that messages higher than log level debug are human-readable. For example this one:

```
                    Log(LOG_LEVEL_ERR, "Invalid ACL in '%s' at index %d (acl_check: %s)"
                        acl_text_str, last, acl_error(retv)); 
```